scripts/rgbColors.txt

```diff
@@ -0,0 +1,76 @@
+--white: #FFFFFFFF;
```

Comment:
do we need this file with hex colors or is it leftover?

Reply:
I left it so it's easier to figure out in the future which values has changed (in figma colors are in hex). Also it should be easier to change and translate values to oklch once again if needed